image_describe.go

```diff
@@ -45,6 +45,8 @@ func runImageDescribe(cli *CLI, cmd *cobra.Command, args []string) error {
 	fmt.Printf("OS flavor:\t%s\n", image.OSFlavor)
 	fmt.Printf("OS version:\t%s\n", na(image.OSVersion))
 	fmt.Printf("Rapid deploy:\t%s\n", yesno(image.RapidDeploy))
+	fmt.Printf("PROTECTION:\n")
```

Comment:
Because I have seen this and without reading further thought this is our default for nested keys :man_facepalming: 
https://github.com/hetznercloud/cli/blob/master/server_describe.go#L120